tests/wpt/metadata/html/semantics/scripting-1/the-script-element/module/dynamic-import/string-compilation-nonce-module.html.ini

```diff
@@ -1,19 +1,7 @@
 [string-compilation-nonce-module.html]
-  [the Function constructor must inherit the nonce from the triggering script, thus execute]
-    expected: FAIL
-
-  [direct eval must inherit the nonce from the triggering script, thus execute]
-    expected: FAIL
-
-  [setTimeout must inherit the nonce from the triggering script, thus execute]
-    expected: FAIL
-
   [inline event handlers triggered via UA code must not inherit the nonce from the triggering script, thus fail]
     expected: FAIL
 
-  [indirect eval must inherit the nonce from the triggering script, thus execute]
-    expected: FAIL
-
   [reflected inline event handlers must not inherit the nonce from the triggering script, thus fail]
     expected: FAIL
```

Comment:
We don't support `nonce` yet; same as `dynamic-import/string-compilation-nonce-classic.html.ini`.